forecast-admin/forecast/opportunities/fixtures/create_offices.py

```diff
@@ -0,0 +1,49 @@
+import json
```

Comment:
One way to simply explain what this does would be to rename this file to: office_fixture_generator.py (which is what I assume this is doing).